### PR TITLE
Prevents Rubocop from inspecting bin directory and schema.rb in spec/internal

### PR DESCRIPTION
**Pull request by flyingzumwalt**

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -5,6 +5,9 @@ AllCops:
   DisplayCopNames: true
   Include:
     - '**/Rakefile'
+  Exclude:
+    - 'spec/internal/bin/*'
```
Comment (grosscol) [inline .rubocop.yml]: Why not just ignore spec/internal/* ?

Comment (flyingzumwalt) [inline .rubocop.yml]: I thought it might be good to catch bad stuff generated into spec/internal by the generators.

Comment (mjgiarlo) [inline .rubocop.yml]: Dig this idea.

**Comment by mjgiarlo** [comment]
The build is failing because Rubocop is now scanning a bunch of stuff in vendor/bundle/. (I'm not sure why the change you made, @flyingzumwalt, would cause so many new Rubocop failures, though...)

**Comment by mjgiarlo** [comment]
:clap: @cbeer 

Exclude the vendor directory also, @flyingzumwalt, and the build should get farther.

**Comment by flyingzumwalt** [comment]
:arrow_up: Green!  @mjgiarlo or @grosscol could you merge?

**Pull request merged**